cmake/modules/FindICU.cmake

```diff
@@ -27,6 +27,7 @@ endforeach()
 
 foreach(sdk ANDROID;FREEBSD;LINUX;WINDOWS)
   foreach(MODULE ${ICU_FIND_COMPONENTS})
+    string(TOUPPER "${MODULE}" MODULE)
```

Comment:
I believe this change has already been made in https://github.com/apple/swift/pull/5432.